tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -225,6 +245,57 @@ def test_file_no_stack_root_match(self, mock_integration):
         assert response.data["error"] == "stack_root_mismatch"
         assert response.data["integrations"] == [serialized_integration(self.integration)]
 
+    @mock.patch.object(ExampleIntegration, "get_blame_for_file")
+    def test_get_commit_sha_valid_line_no(self, mock_blame):
+        mock_blame.return_value = git_blame
+        project_stacktrace_link_endpoint = ProjectStacktraceLinkEndpoint()
+        commit_sha = project_stacktrace_link_endpoint.get_commit_sha(
+            self.integration, self.organization.id, 26, "test/path/file.py", self.repo, "main"
```

Comment:
Could you please kwargs?
When I read the test I do not know right away what `26` refers to.